requirements.txt

```diff
@@ -2,7 +2,7 @@ numpy==1.23.0
 pandas==1.4.3
 pandas-ta==0.3.14b
 
-ccxt==1.89.96
+ccxt==1.90.38
```

Comment:
The dependency-bump breaks OKX (https://github.com/ccxt/ccxt/issues/14284) - as such this cannot currently be merged, as it'd be "one over the other" (introducing a new known bug by fixing another bug).
Fix in ccxt should usually not take too long though.

Reply:
Ok, let's wait for the CCXT fix